src/harness/fourslashImpl.ts

```diff
@@ -4510,6 +4510,32 @@ export class TestState {
 
         this.verifyCurrentFileContent(newFileContent);
     }
+
+    public baselineMapCode(
+        { fileName, pos }: Range,

```

Comment:
You should be using pos and end both right since thats the focus location marked using range. 
We need test cases where there are multiple focus locations with different lengths and as mentioned in API with different priorities too.

Reply:
the current code mapper only uses the start location. I can add a test for multiple focusLocations, though.

Reply:
I've added a test to cover this, `mapCodeReplaceUsingRange`